addons/website_sale/models/sale_order.py

```diff
@@ -166,6 +176,19 @@ def _cart_accessories(self):
             accessory_products -= order.website_order_line.mapped('product_id')
             return random.sample(accessory_products, len(accessory_products))
 
+    def action_mark_as_paid(self):
```

Comment:
Pay attention to code previously added in website_portal_sale then re-moved here: just move it directly to the right place to avoid diff.